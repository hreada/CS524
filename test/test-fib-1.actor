rec(lambda(b, lambda(n,
  if(le(n, 2), 1, add(app(b, minus(n, 1)),
                      app(b, minus(n, 2)))))))

