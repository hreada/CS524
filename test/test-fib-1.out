<T>
  <actors>
    <actor>
      <k> exec ( lambda ( n , if ( le ( n , 2 ) , 1 , add ( app ( lambda ( _2 , app ( app ( lambda ( _1 , app ( lambda ( b , lambda ( n , if ( le ( n , 2 ) , 1 , add ( app ( b , minus ( n , 1 ) ) , app ( b , minus ( n , 2 ) ) ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) , lambda ( _1 , app ( lambda ( b , lambda ( n , if ( le ( n , 2 ) , 1 , add ( app ( b , minus ( n , 1 ) ) , app ( b , minus ( n , 2 ) ) ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) ) , _2 ) ) , minus ( n , 1 ) ) , app ( lambda ( _2 , app ( app ( lambda ( _1 , app ( lambda ( b , lambda ( n , if ( le ( n , 2 ) , 1 , add ( app ( b , minus ( n , 1 ) ) , app ( b , minus ( n , 2 ) ) ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) , lambda ( _1 , app ( lambda ( b , lambda ( n , if ( le ( n , 2 ) , 1 , add ( app ( b , minus ( n , 1 ) ) , app ( b , minus ( n , 2 ) ) ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) ) , _2 ) ) , minus ( n , 2 ) ) ) ) ) ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) </definedaddr>
</T>
